### PR TITLE
feat: [PIE-4708]: pipeline list view VQA (Visual Quality Assurance) changes

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

Made visual changes as per https://www.figma.com/file/4HavSweFhZeVsJoaWwSrj8/Pipelines?node-id=2333%3A133063

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 3 Issues 
* Code issue 3 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10433)

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Review by satyam-harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.tsx [hunk 1]
```diff
@@ -71,14 +71,14 @@ export function PipelineListTable({
...
+        width: '12%',
         disableSortBy: true,
         Cell: CodeSourceCell
       },
       {
         Header: getString('pipeline.recentExecutions').toUpperCase(),
         accessor: 'recentExecutions',
-        width: '25%',
+        width: '28%',
         Cell: RecentExecutionsCell,
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.tsx:82]: Will these numbers work on bigger screens as well?

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good, with one comment
https://github.com/harness/harness-core-ui/pull/10433/files#r938864955

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.tsx (hunk 1 above)
Comment (udayvunnam) [reply src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.tsx:82]: Yes, they are relative. However, we can move to a grid layout for the table in the future where we can represent the columns in a better way.

**Pull request merged**